### PR TITLE
Add rac_textColor support

### PR DESCRIPTION
**Pull request by patrickreynolds**

### CODE REVIEW

File: ReactiveBind/ViewBinding.swift [hunk 1]
```diff
@@ -43,6 +43,10 @@ extension UILabel {
     public var rac_attributedText: MutableProperty<NSAttributedString?> {
         return lazyMutableProperty(self, &ReactiveBindAssocationKeys.AttributedTextProperty, { self.attributedText = $0 }, { self.attributedText  })
     }
+
+    public var rac_textColor: MutableProperty<UIColor?> {
+        return lazyMutableProperty(self, &ReactiveBindAssocationKeys.TextColorProperty, { self.textColor = $0 }, { self.text })
```
Comment (mpurland) [inline ReactiveBind/ViewBinding.swift]: getter should return self.textColor

Comment (patrickreynolds) [inline ReactiveBind/ViewBinding.swift]: Good catch :+1:

**Comment by mpurland** [comment]
Looks good.

**Comment by mpurland** [comment]
:+1: :boat:

**Pull request merged**